fyle_accounting_mappings/models.py

```diff
@@ -703,3 +706,46 @@ def create_or_update_category_mapping(
         )
 
         return category_mapping
+
+    @staticmethod
+    def bulk_create_mappings(destination_attributes: List[DestinationAttribute],
+                             destination_type: str, workspace_id: int, set_auto_mapped_flag: bool = True):
+        """
+        Create the bulk mapping
+        :param destination_attributes: Destination Attributes List with category mapping as null
+        """
+        attribute_value_list = []
+
+        for destination_attribute in destination_attributes:
+            attribute_value_list.append(destination_attribute.value)
+
+        # Filtering unmapped Expense Attributes
+        source_attributes = ExpenseAttribute.objects.filter(
+            workspace_id=workspace_id,
+            attribute_type='CATEGORY',
+            value__in=attribute_value_list,
+            categorymapping__source_category__isnull=True
+        ).values('id', 'value')
+
+        source_attributes_id_map = {source_attribute['value'].lower(): source_attribute['id'] \
+            for source_attribute in source_attributes}
+
+        mapping_creation_batch = []
+
+        for destination_attribute in destination_attributes:
+            if destination_attribute.value.lower() in source_attributes_id_map:
+                destination = {}
+                if destination_type == 'EXPENSE_TYPE':
```

Comment:
netsuite is not handled here, in netsuite it will be EXPENSE_CATEGORY, pls verify. Check if netsuite code matches 100% to this.

Reply:
it is EXPENSE_CATEGORY 😛

Reply:
Wait it is `EXPENSE_TYPE` only for Intacct. 
For Netsuite it is `EXPENSE_CATEGORY`

Reply:
that's what I've mentioned above no?